File: src/ModuleFrontend/ModuleFrontend.Api/Models/Module.cs
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;

namespace ModuleFrontend.Api.Models
{
    [ExcludeFromCodeCoverage]
    public class Module
    {
        public long ModuleId { get; set; }
        public string ModuleNaam { get; set; }
        public string ModuleCode { get; set; }
        public string Cohort { get; set; }
        public int AantalEc { get; set; }
        public string Studiejaar { get; set; }
        public Moduleleider Moduleleider { get; set; }
        public Studiefase Studiefase { get; set; }
        public IEnumerable<Specialisatie> VerplichtVoor { get; set; }
        public Matrix Competenties { get; set; }
        public IEnumerable<Specialisatie> AanbevolenVoor { get; set; }
        public List<string> Eindeisen { get; set; }
    }
}

File: src/ModuleFrontend/ModuleFrontend.Api/ClientApp/src/app/models/module.ts
import { Moduleleider } from "./module-leider";
import { Studiefase } from "./studiefase";
import { Specialisatie } from "./specialisatie";
import { Matrix } from "./matrix";
import { Cohort } from "./cohort";

export interface Module {
    id: number
    moduleNaam: string
    moduleCode: string
    cohort: Cohort,
    aantalEc: number
    studiejaar: string
    moduleleider: Moduleleider
    studiefase: Studiefase
    verplichtVoor: Specialisatie[]
    aanbevolenVoor: Specialisatie[]
    competenties: Matrix
    beschrijvingLeerdoelen: string
    inhoudelijkeBeschrijving: string
    eindeisen: string
    contacturenWerkvormen: string
    toetsvorm: string
    voorwaardenVoldoende: string
    letOp: string
    summatief: boolean
    formatief: boolean
    kwalitatief: boolean
    kwantitatief: boolean
    examinatoren: string
}
